Formula/riff.rb

```diff
@@ -0,0 +1,32 @@
+require "open3"
+
+class Riff < Formula
+  desc "Function As A Service on top of Knative, riff is for functions"
+  homepage "https://www.projectriff.io/"
+  url "https://github.com/projectriff/riff.git",
+      :tag      => "v0.2.0",
+      :revision => "1ae190ff3c7edf4b375ee935f746ebfd1d8eaf5c"
+
+  depends_on "go" => :build
+  depends_on "make" => :build
```

Comment:
Can't it install with macOS system make?

Reply:
Oops, I didn't realize it was shipped by default. Let me remove the dependency.

Reply:
Done